project-base/app/config/graphql/types/ModelType/Payment/PaymentStatus.types.yaml

```diff
@@ -0,0 +1,10 @@
+PaymentStatus:
+    type: object
+    config:
+        fields:
+            isPaid:
+                type: "Boolean!"
+                description: "Whether the order is already paid or not"
+            transactionCount:
+                type: "Int!"
+                description: "Count of already processed transactions"
```

Comment:
I can't tell. It's no problem to add the field with the max count, but I'm not sure if it's useful on the storefront. 

@sebaholesz can you please let me know whether I should implement this?

Reply:
It is, @sebaholesz has already implemented his own constant in order-payment-confirmations.tsx for this purpose, so I think it would be better to set it in one place and sent via API

Reply:
Yeah, it would make sense. Having it in two places seems not correct.

Reply:
done in https://github.com/shopsys/shopsys/pull/2876/commits/c06e35f10f18b88a67a6c85ad4dd1a2e1b3a4dc6
@TomasLudvik @sebaholesz could you please review it?

Reply:
done in c86b1abc8
@sebaholesz @TomasLudvik could you please review it?